UnitySDK/Assets/ML-Agents/Scripts/Grpc/RpcCommunicator.cs

```diff
@@ -211,6 +211,10 @@ void SendCommandEvent(CommandProto command)
                     }
                 case CommandProto.Reset:
                     {
+                        foreach (var brainName in m_ActionCallbacks.Keys)
```

Comment:
Can you just replace this (and the other loop) with `m_ActionCallbacks.Clear()`?